collectors/node/cpu.go

```diff
@@ -167,3 +167,11 @@ func round(f float64) float64 {
 	shift := math.Pow(10, float64(2))
 	return math.Floor(f*shift+.5) / shift
 }
+
+// Helper function: replace all negative numbers with zero
+func gtZero(n float64) float64 {
+	if math.Signbit(n) {
```

Comment:
yes, Signbit shifts left 63 bits and check that the most significant bit is set. https://golang.org/src/math/signbit.go?s=233:261#L1

i don't think you really want this